tests/Unit/Domains/Contact/ManageReminders/Services/UpdateContactReminderTest.php

```diff
@@ -33,6 +33,44 @@ public function it_updates_a_reminder(): void
         $this->executeService($regis, $regis->account, $vault, $contact, $reminder);
     }
 
+    /** @test */
+    public function it_persists_notification_channels(): void
+    {
+        $regis = $this->createUser();
+        $vault = $this->createVault($regis->account);
+        $vault = $this->setPermissionInVault($regis, Vault::PERMISSION_EDIT, $vault);
+        $contact = Contact::factory()->create(['vault_id' => $vault->id]);
+        $reminder = ContactReminder::factory()->create([
+            'contact_id' => $contact->id,
+        ]);
+
+        UserNotificationChannel::factory()->create([
+            'user_id' => $regis->id,
+            'preferred_time' => '18:00',
+        ]);
+
+        $request = [
+            'account_id' => $regis->account->id,
+            'vault_id' => $vault->id,
+            'author_id' => $regis->id,
+            'contact_id' => $contact->id,
+            'contact_reminder_id' => $reminder->id,
+            'label' => 'birthdate',
+            'day' => 29,
+            'month' => 10,
+            'year' => 1981,
+            'type' => ContactReminder::TYPE_ONE_TIME,
+            'frequency_number' => null,
+        ];
+
+        (new UpdateContactReminder())->execute($request);
+        (new UpdateContactReminder())->execute($request);
```

Comment:
I'm not entirely sure why, but in the context of the unit test, the old implementation would fail only if `UpdateContactReminder` was executed twice